File: synthetic_test/syn.py
import os

def main() -> None:
    iters: int = int(os.environ["SPEEDTEST_ITERS"])
    for i in range(iters):
        a: list[int] = [0] * 1000

        for j in range(100):
            a[j] = j

        for j in range(10):
            tmp: int = a[j]
            a[j] = a[1000 - (10- j)]
            a[1000 - (10- j)] = tmp


if __name__ == "__main__":
    raise SystemExit(main())

File: synthetic_test/syn.swift
import Foundation

if let itersString = ProcessInfo.processInfo.environment["SPEEDTEST_ITERS"],
   let iters = Int(itersString) {
    for _ in 1...iters {
        var lst = Array(repeating: 0, count: 1000)

        for j in 1..<1000 {
            lst[j] = j
        }

        for j in 1...10 {
            let tmp = lst[j]
            lst[j] = lst[999 - (10 - j)]
            lst[999 - (10 - j)] = tmp
        }
    }
}
